### ISSUE TITLE
Adding Expr

### ISSUE THREAD

**Issue by superfunc** [ku-fpg/wakarusa#1]
I was looking to work some Expr stuff in here, could we consolidate the repo to have one src folder?

**Comment by andygill** [comment]
Yes. I'm trying to clean up the various ideas into one place right now.
